atom/common/api/event_emitter_caller.cc

```diff
@@ -20,8 +20,12 @@ v8::Local<v8::Value> CallMethodWithArgs(v8::Isolate* isolate,
                                    v8::MicrotasksScope::kRunMicrotasks);
   // Use node::MakeCallback to call the callback, and it will also run pending
   // tasks in Node.js.
-  return node::MakeCallback(isolate, obj, method, args->size(), &args->front(),
-                            {0, 0}).ToLocalChecked();
+  v8::MaybeLocal<v8::Value> ret = node::MakeCallback(isolate, obj, method, args->size(), &args->front(),
```

Comment:
@MarshallOfSound The linter doesn't seem to like the length of this line.

Reply:
Fixed 👍